examples/api/lib/widgets/routes/flexible_route_transitions.0.dart

```diff
@@ -0,0 +1,228 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
```

Comment:
Include a blurb here about what this example shows, for example: https://github.com/flutter/flutter/blob/d1385a48271f95356c2f186f5a45076a2c146be2/examples/api/lib/widgets/navigator_pop_handler/navigator_pop_handler.0.dart#L7-L9

Reply:
This is done below now :+1: